tools/docker-compose/Dockerfile

```diff
@@ -46,6 +46,7 @@ RUN yum -y install acl \
   nodejs \
   openldap-devel \
   openssh-server \
+  patch \
```

Comment:
What's this about?

Reply:
Someone recently added a helpful updater.sh script to the requirements/ directory, but it requires `patch` to be installed.